documentation/guides/consumer/basic_setup.md

```diff
@@ -33,5 +35,5 @@ end
 - durable topic `example_exchange` exchange created or redeclared
 - durable `example_queue` queue created or redeclared and bound to `example_exchange` exchange
 - queue `example_queue` has a deadletter exchange set to `example_exchange.deadletter`
-- every `handle_message` callback will be executed in a separate process
+- every `handle_message` callback will be executed in a separate process (supervised task)
```

Comment:
Is "supervised task" a fixed term in Elixir?
Do we need to link to definition of this, to make it clear what we are talking about?

Reply:
A link would probably be good yes! https://hexdocs.pm/elixir/1.10.3/Task.html#module-supervised-tasks

Reply:
Thanks for the link. See separate thread with suggestion.